configure.ac

```diff
@@ -335,6 +335,7 @@ OPTION_DEFAULT_DISABLE([app-sampler], [ENABLE_APP_SAMPLER])
 OPTION_DEFAULT_DISABLE([store-app], [ENABLE_STORE_APP])
 OPTION_DEFAULT_DISABLE([test_sampler], [ENABLE_TEST_SAMPLER])
 OPTION_DEFAULT_DISABLE([list_sampler], [ENABLE_LIST_SAMPLER])
+OPTION_DEFAULT_DISABLE([record_sampler], [ENABLE_RECORD_SAMPLER])
```

Comment:
@narategithub IMO, besides explicitly giving --enable-list_sampler or --enable-record_sampler, they would be enabled when --enable-ldms-test is given. This will be consistent with the condition to enable test_sampler. What do you think?